lib/alamo.js

```diff
@@ -101,11 +103,6 @@ async function get_region_name_by_space(pg_pool, space_name) {
   return info.region_name
 }
 
-async function get_region_api_by_space(pg_pool, space_name) {
-  let info = await fetch_api_by_space(pg_pool, space_name)
-  return info.stack_api
-}
-
 async function get_region_api_by_space(pg_pool, space_name) {
```

Comment:
duplicate function, not sure how this redefinition was allowed in node 8 but not node 10